config/foreman-answers.yaml

```diff
@@ -70,3 +70,6 @@ foreman_proxy::plugin::salt: false
 foreman_proxy::plugin::shellhooks: false
 puppet:
   server: true
+  server_jvm_extra_args:
```

Comment:
Are there any other commonly configured jvm settings that users would pass to this argument and potentially wipe out these defaults without realizing what they have done?

Reply:
The migration should take care of that. These are just the defaults for fresh installs.

Reply:
I was meaning more from a user passing arguments to the installer *after* a fresh install.